galaxy_ng/app/management/commands/set-repo-keyring.py

```diff
@@ -55,7 +56,7 @@ def handle(self, *args, **options):
 
         if not options["yes"]:
             confirm = input(
-                f"This will set keyring to {keyring} for {repository} repository, " "Proceed? (Y/n)"
+                f"This will set keyring to {keyring_path} for {repository} repository, " "Proceed? (Y/n)"
```

Comment:
It's my understanding that `keyring` and `keyring_path` both indicate the string `/etc/pulp/certs/galaxy.kbx`